<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="posWidget">
     <attribute name="title">
      <string>POS</string>
     </attribute>
     <widget class="QTabWidget" name="posTab">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>551</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="newInvoiceTab">
       <attribute name="title">
        <string>New Invoice</string>
       </attribute>
       <widget class="QComboBox" name="nameComboBoxNewInvoiceTab">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>30</y>
          <width>421</width>
          <height>22</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="maxVisibleItems">
         <number>5</number>
        </property>
       </widget>
       <widget class="QComboBox" name="productComboBoxNewInvoiceTab">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>110</y>
          <width>281</width>
          <height>22</height>
         </rect>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentText">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="emailLineEditNewInvoiceTab">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>70</y>
          <width>281</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="nameLabelNewInvoiceTab">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
       <widget class="QLabel" name="emailLabelNewInvoiceTab">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
       <widget class="QLabel" name="idLabelNewInvoiceTab">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>70</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="idLineEditNewInvoiceTab">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>70</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="productLabelNewInvoiceTab">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Product</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="productNumberSpinBoxNewInvoiceTab">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>140</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="addProductButtonNewInvoiceTab">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>140</y>
          <width>161</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Add Product To Invoice</string>
        </property>
       </widget>
       <widget class="QPushButton" name="removeProductButtonNewInvoiceTab">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>180</y>
          <width>161</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Remove Product From Invoice</string>
        </property>
       </widget>
       <widget class="QPushButton" name="purchaseButtonNewInvoiceTab">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>220</y>
          <width>281</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Purchase</string>
        </property>
       </widget>
       <widget class="QLabel" name="invoiceTotalLabelNewInvoiceTab">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>470</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Invoice Total</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="invoiceTotalLineEditNewInvoiceTab">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>470</y>
          <width>261</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableWidget" name="invoiceListTableWidgetNewInvoiceTab">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>30</y>
          <width>256</width>
          <height>421</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="addCustomerTab">
       <attribute name="title">
        <string>Add Customer</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="editCustomerTab">
       <attribute name="title">
        <string>Edit Customer</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="adminWidget">
     <attribute name="title">
      <string>Admin</string>
     </attribute>
     <widget class="QTabWidget" name="adminTab">
      <property name="geometry">
       <rect>
        <x>-4</x>
        <y>-1</y>
        <width>801</width>
        <height>551</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="addInventoryTab">
       <attribute name="title">
        <string>Add Inventory</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="editInventoryTab">
       <attribute name="title">
        <string>Edit Inventory</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="addVendorTab">
       <attribute name="title">
        <string>Add Vendor</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="editVendorTab">
       <attribute name="title">
        <string>Edit Vendor</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="shipmentsTab">
       <attribute name="title">
        <string>Shipments</string>
       </attribute>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
